airbyte-integrations/connectors/destination-aws-datalake/destination_aws_datalake/spec.json

```diff
@@ -90,31 +90,40 @@
         "description": "The region of the S3 bucket. See <a href=\"https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions\">here</a> for all region codes.",
         "enum": [
           "",
-          "us-east-1",
```

Comment:
Sometimes there's a leading "", in the SPEC options, sometimes there isn't - I left things as they were.